packages/node_modules/@node-red/editor-api/lib/admin/flows.js

```diff
@@ -55,7 +55,16 @@ module.exports = {
             if (version === "v1") {
                 opts.flows = {flows: req.body}
             } else {
-                opts.flows = req.body;
+                var body = req.body;
+                if (false && body.hasOwnProperty("flows")) {
```

Comment:
Looks like the `false &&` option has been left in by mistake?

Reply:
Having looked more closely, I can see none of the changes to this file are needed.

`opts.flows` is set to `req.body` which is an object containing the flows and maybe the credentials properties. No need for this logic to pull credentials out separately.